tools/hangar/__snapshots__/test_corpus/sdk_tests/function/aws-function.test.w_compile_tf-aws.md

```diff
@@ -184,17 +111,6 @@
       }
     },
     "aws_s3_object": {
-      "FunctionAccessingContext_S3Object_FCA6F30A": {
-        "//": {
-          "metadata": {
-            "path": "root/Default/Default/FunctionAccessingContext/S3Object",
-            "uniqueId": "FunctionAccessingContext_S3Object_FCA6F30A"
-          }
-        },
-        "bucket": "${aws_s3_bucket.Code.bucket}",
-        "key": "<ASSET_KEY>",
-        "source": "<ASSET_SOURCE>"
-      },
```

Comment:
Are all these objects not needed anymore because of the string to json change? 🤔

Reply:
I think this is just because I refactored the test a little bit - but yeah otherwise this would be pretty odd